### PR TITLE
Added test case for Sol Ex Settings banner

### PR DESCRIPTION
**Pull request by tonyxrmdavidson**
# Description
New test-case for sol ex settings

### CODE REVIEW

File: test-cases/tests/walkthroughs/g16-verify-settings-notifications-pop-up.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+estimate: 15m
+tags:
```
Comment (trepel) [inline test-cases/tests/walkthroughs/g16-verify-settings-notifications-pop-up.md:3]: should be `targets`, not `tags` I think

File: test-cases/tests/walkthroughs/g16-verify-settings-notifications-pop-up.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - 2.6.0
+---
+
+# G16 - Verify settings notifications pop up
+
+Acceptance Criteria:
+
+When logged into solution explorer as a customer-admin
```
Comment (trepel) [inline test-cases/tests/walkthroughs/g16-verify-settings-notifications-pop-up.md:11]: not `customer-admin`, but `dedicated-admin` ? Even more precise would be to say `a user from didicated-admin group` but that's probably an overkill

File: test-cases/tests/walkthroughs/g16-verify-settings-notifications-pop-up.md (hunk 1 above)
Comment (tonyxrmdavidson) [reply test-cases/tests/walkthroughs/g16-verify-settings-notifications-pop-up.md]: Yeah, you're correct.  Fixed it with new commit.

File: test-cases/tests/walkthroughs/g16-verify-settings-notifications-pop-up.md (hunk 2 above)
Comment (tonyxrmdavidson) [reply test-cases/tests/walkthroughs/g16-verify-settings-notifications-pop-up.md]: Agree, updated with commit

**Pull request merged**